### PR TITLE
Update `Data Centers` documentation

### PR DESCRIPTION
**Pull request by Gabeblis**
## Description
Updated `Data Centers` documentation to account for a bug found with the current implementation of the data-center props.

### CODE REVIEW

**Comment by Gabeblis** [comment]
@aj-stein-gsa @kyhu65867 When you get a chance, please give this a quick review to make sure it aligns with the solution for the bug found in https://github.com/GSA/fedramp-automation/issues/818.

**Comment by Gabeblis** [comment]
> This looks good, but can adjust the feedback referenced in the tracking release PR into main in this PR here.
> 
> [#45 (comment)](https://github.com/GSA/automate.fedramp.gov/pull/45#discussion_r1814970711)

Done: 7f31e8590608235b8ca3b8f7bbe4e7427985f9c4

**Review by aj-stein-gsa — APPROVED** [review]
Thank you sir, looks good to me.

**Review by Rene2mt — APPROVED** [review]

**Review by wandmagic — APPROVED** [review]

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 1]
```diff
@@ -744,7 +744,7 @@ Each system must define at least two data centers. There must be exactly one pri
           <postal-code>00000</postal-code>
           <country>US</country>
        </address>
-       <prop name="data-center" value="some-location-value" class="primary"/>
+       <prop name="type" class="primary" value="data-center"/>
```
Comment (kyhu65867) [inline content/documentation/ssp/4-ssp-template-to-oscal-mapping.md:747]: I know order doesn't matter, but wondering if it makes sense to put value ahead of class. (Not sure why, I think my head was putting type and data center together as opposed to type and primary. But I don't think this is a big deal)

**Review by kyhu65867 — COMMENTED** [review]
Wanted to know your thoughts prior to approval.

**Comment by kyhu65867** [comment]
(Whoops I didn't realize this was merged already disregard)

**Pull request merged**